eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -195,6 +195,17 @@ jobs:
                 sourceFolder: build/sdk/${{ parameters.ServiceDirectory }}/${{ artifact.Path }}/docs/html
                 targetFolder: $(Build.ArtifactStagingDirectory)/docs/${{ artifact.Name }}
               displayName: Copy documentation to artifact staging directory
+             
+            - task: Powershell@2
+              inputs:
+                filePath: $(Build.SourcesDirectory)/eng/common/scripts/Verify-ChangeLog.ps1
+                arguments: >
+                  -ChangeLogLocation sdk/${{ parameters.ServiceDirectory }}/${{ artifact.Path }}/CHANGELOG.md
+                  -VersionString (Get-Content build/sdk/${{ parameters.ServiceDirectory }}/${{ artifact.Path }}/docs/html/package-info.json | ConvertFrom-Json).version
+                  -ForRelease $true
```

Comment:
Do not specify `-ForRelease` here. This will do some basic validation (including formatting and version number matching) but won't do all validation.

Reply:
@gearama Did you guys decide to keep the ForRelease check in all cases? I think that may cause some issues for normal pipelines that aren't ready to release yet. For example this will enforce that your latest entry always has a release date which I suspect you don't want in normal cases.